client/src/components/SearchPage/SearchField.js

```diff
@@ -0,0 +1,111 @@
+import React, {Component} from 'react';
+import './SearchField.css'
+import Logo from '../assets/logo.png'
+import Search from '../assets/search.png';
+import Mic from '../assets/mic.png'
+import Suggestion from "./data";
+import data from './data'
+class SearchField extends Component{
+
```

Comment:
Bruh. Code style. 

What are all the newlines for? Remove them. Follow proper style. Use a linter if you have to (I'd recommend Prettier - it works pretty well with React)